lib/bap_llvm/bap_llvm_binary.ml

```diff
@@ -63,3 +63,6 @@ external section_addr : t -> int -> int64  =
 
 external section_size : t -> int -> int64  =
   "bap_llvm_binary_section_size_stub"
+
+external llvm_load : Bigstring.t -> string =
```

Comment:
prefix our external symbols with `bap` we are not the LLVM team, we are the BAP Team. The chances to get a name collision with other `llvm` function are high.